### ISSUE TITLE
Train script

### ISSUE THREAD

**Issue by anjany** [lisha-chen/Deep-structured-facial-landmark-detection#3]
Hello @lisha-chen : Given that the training schedule (Algo 1) are crucial for your method, do you plan to make the training script available? (If I have missed it, could you please point me to it?). Thank you!

**Comment by lisha-chen** [comment]
Hello, thank you for your interest. The crucial part is the loss functions used, provided in CNN_models/FAN_crf.py. 
For your convenience, refer to the function "crf_forward" in CNN_models/FAN_crf.py, where you can directly use the function and "self.loss_unary" is for pre-training the CNN model, and the "self.loss_joint" is for jointly training the CNN-CRF model. Using tensorflow to take gradient w.r.t. the parameters of interest in each stage, you can follow the paper to write the training script.

**Comment by anjany** [comment]
Thanks for the hints, Lisha. Let me give this a try. I'll close this ticket for now. I hope it is okay for me to correspond with you on this issue here.
